src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.pip.account.management.service;
+
+
+import lombok.extern.slf4j.Slf4j;
+import net.minidev.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientException;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@Slf4j
+@Component
+public class PublicationService {
+    @Value("${service-to-service.publication-services}")
+    private String url;
+
+
+    public String sendNotificationEmail(String emailData, String forename, String surname) {
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put("email", emailData);
+        jsonObject.put("forename", forename);
+        jsonObject.put("surname", surname);
+        jsonObject.put("isExisting", false);
+        WebClient webClient = WebClient.create();
+        log.info("Attempting to send email to " + url);
```

Comment:
this log could go, doesnt really provide much value by logging the url

Reply:
sorted